File: New folder/notes.ml
(* ocaml notes *)
let greet name = "hello " ^ name

let () =
  print_endline (greet "practice");
  let nums = [1; 2; 3] in
  let doubled = List.map (fun n -> n * 2) nums in
  List.iter (fun n -> print_int n; print_newline ()) doubled

(* TODO: try pattern matching next *)

File: New folder/notes.scala
// scala notes
object Notes {
  def greet(name: String): String = s"hello $name"

  def main(args: Array[String]): Unit = {
    println(greet("practice"))

    val nums = List(1, 2, 3)
    val doubled = nums.map(_ * 2)
    println(doubled)
  }
}
// TODO: try pattern matching next
